### PR TITLE
chore: Add inviter name to invite email subject

### LINKED ISSUE #8949: Show who is inviting you to Parabol
@garethaledavies suggested [here](https://parabol.slack.com/archives/C03NGEJUKCN/p1696672232145009) that we should show who is inviting you to Parabol in the subject line. 

At the moment, it says, "You've been invited to Parabol", which one might think it spam.

### AC

- [ ] Email subject line says who has invited you to Parabol

### PR DESCRIPTION
**Pull request by Mohd-Muneeb**
# Description

Fixes/Partially Fixes #8949

Updated the message subject sent to the invitee by the inviter as mentioned in the issue.

## Demo

Unavailable

## Testing scenarios

No tests were performed, although type safety was ensured with standard measures

### CODE REVIEW

File: packages/server/email/teamInviteEmailCreator.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import TeamInvite, {TeamInviteProps} from 'parabol-client/modules/email/componen
 import {headCSS} from 'parabol-client/modules/email/styles'
 import React from 'react'
 
-const subject = 'You’ve been invited to Parabol'
+const subject = (inviterEmail: string) : string => `${inviterEmail} has invited you to Parabol`
```
Comment (Dschoordsch) [inline packages/server/email/teamInviteEmailCreator.tsx:6]: -1 I think the inviter name would be nicer to read here

**Review by Dschoordsch — CHANGES_REQUESTED** [review]
Thanks for looking into this, could you make the small change?

File: packages/server/email/teamInviteEmailCreator.tsx (hunk 1 above)
Comment (Mohd-Muneeb) [reply packages/server/email/teamInviteEmailCreator.tsx]: Resolved the request!

**Review by Dschoordsch — APPROVED** [review]

**Comment by Dschoordsch** [comment]
Thanks for the fix!

**Comment by Mohd-Muneeb** [comment]
:D

**Pull request merged**